### PR TITLE
Update the Gemfile.lock for security updates.

### PR DESCRIPTION
**Pull request by dperez37**
This PR updates the gemfile.lock file to get security updates.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -11,7 +11,7 @@ GEM
     colored2 (3.1.2)
     cork (0.3.0)
       colored2 (~> 3.1)
-    danger (8.5.0)
+    danger (8.6.1)
```
Comment (byencho) [inline Gemfile.lock:14]: Do we need to update all of these or is there a specific dependency we can just update?

Comment (dperez37) [reply Gemfile.lock:14]: Octokit is the main one but I usually update everything when I do this stuff.

Comment (byencho) [reply Gemfile.lock:14]: OK. If that's what you've already been doing.

**Review by byencho — APPROVED** [review]

**Comment by dperez37** [comment]
Thanks @byencho

**Pull request merged**